Brandon Benefield & Il Ho Kim

bbenefield89 opened the pull request.

bbenefield89 committed:
RENAME `dotenv`

- in dotenv
rename `dotenv` to `.env`

bbenefield89 committed:
ADD NODEMON

- in package.json

bbenefield89 committed:
GENERATE KEYPAIR FROM `nacl`

- in app.js
use `nacl.box.keyPair()` to generate new public & secret keys

bbenefield89 committed:
DECODE GIST CONTENT

- in app.js
`/createsecret` route decodes UTF8 gists into Base64 unicode

nikhilkamineni commented:
Nice progress so far! I would double check your `/createsecret` endpoint though. It doesn't look like you're encrypting your message so that it will be able to be decrypted correctly later (hint: checkout the `secretbox` part of the tweetnacl docs)

Commit:
feat: add 'Create an encrypted gist for yourself' from Sean's lecture

bbenefield89 committed:
GENERATE SECRET KEY AND DISPLAY

- in app.js
generate secret key or read from `config.json` and display it to the user

bbenefield89 committed:
CAN VIEW LIST OF USERS GISTS

- in app.js
route GET:`/gists`

Commit:
feat: retrieve encrypted GIST

bbenefield89 committed:
Merge pull request #1 from bbenefield89/brandons

Il Ho Push

Commit:
feat: add encrypt data from '/fetchmessagefromself:id'
